src/components/advanced/ReturnToSearchButton/ReturnToSearchButton.tsx

```diff
@@ -1,11 +1,14 @@
-import styles from './ReturnToSearchButton.module.scss';
-import { LinkWithQuery } from '../../common/LinkWithQuery/LinkWithQuery';
+'use client'
+import styles from '@components/advanced/ReturnToSearchButton/ReturnToSearchButton.module.scss';
+import { LinkWithQuery } from '@components/common/LinkWithQuery/LinkWithQuery';
 import React from 'react';
 
 export default function ReturnToSearchButton() {
   return (
       <button aria-label="Return to Search" className={styles.searchIcon} data-testid="return-to-search">
-        <LinkWithQuery to="/"><i className="fa-solid fa-magnifying-glass"></i></LinkWithQuery>
+        <LinkWithQuery to="/">
+          <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352a144 144 0 1 0 0-288 144 144 0 1 0 0 288z"/></svg>
```

Comment:
Could you please move this and other svgs to their separate components?

Reply:
Moved all svgs to their separate components